lib/main.dart

```diff
@@ -40,6 +38,9 @@ class App extends StatelessWidget {
       providers: [
         BlocProvider(create: (_) => AppTheme()),
         BlocProvider(create: (_) => NavBarIndex()),
+        BlocProvider(create: (_) => MobileEditing()),
+        BlocProvider(create: (_) => ValidSurveyQuestions()),
+        BlocProvider(create: (_) => ValidSurveyAnswers()),
```

Comment:
I really wanted these BLoCs to be scoped to only the survey routes, but apparently a `ReorderableList` takes a widget out of its normal context while it's being reordered, so I had to move these providers up the heirarchy.